stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -65,6 +68,8 @@
   private ChildNodeRemovedEvent mCachedChildNodeRemovedEvent;
   private ChildNodeInsertedEvent mCachedChildNodeInsertedEvent;
 
+  private final Map<String, List<Integer>> mSearchResults;
```

Comment:
Doesn't seem like ManagedIntArray is used anywhere else in the project. Is it part of the Java standard library? Is [this](http://www.blazegraph.com/docs/api/com/bigdata/rdf/graph/impl/util/ManagedIntArray.html) the class?